lib/postgrex/extensions/binary.ex

```diff
@@ -373,6 +382,23 @@ defmodule Postgrex.Extensions.Binary do
     <<value_byte_size::int32>> <> value
   end
 
+  defp encode_network(%Postgrex.INET{address: {_, _, _, _} = addr}),
```

Comment:
Oh, I see, I was confused because it wasn't grouped on struct. You don't have to change but you can alternatively add a `tuple_size` guard.